docs/schema/reference.md

```diff
@@ -248,7 +248,7 @@ The award section is used to announce any awards issued for this tender. There c
 
 ### Contract
 
-The contract section is used to provide details of contracts that have been entered into. Every contract must have a related award, linked via the `awardID` field. This is because supplier information is contained within the 'award'. 
+The `contracts` array is used to provide details of contracts that have been entered into. Every contract must have a related award, linked via the `awardID` field. This is because supplier information is contained within the 'award'. Each contract is detailed using the `Contract` subschema.
```

Comment:
```suggestion
The `contracts` array is used to provide details of contracts that have been entered into. Every contract must have a related award, linked via the `awardID` field. This is because supplier information is contained within the 'award'.
```

Reply:
again I think this is needed otherwise the actual subschema that contains the data fields isn't being mentioned anywhere